packages/alt-ergo/alt-ergo.0.95.2/opam

```diff
@@ -1,24 +1,34 @@
+(******************************************************************************
+* Please, do not modify this file without contacting the maintainers
+* to discuss the reasons of the modification.
+* In particular, dev-repo is not provided because it's private.
+******************************************************************************)
```

Comment:
that is the general case for `opam-repository`? (at least how I understand it; maybe opam-repo admins have a concrete opinion on that @samoht @avsm @dsheets).  I think having such a disclaimer in every opam file is a really bad idea.